src/applications/vaos/tests/new-appointment/components/ReasonForAppointmentPage.unit.spec.js

```diff
@@ -282,27 +282,26 @@ describe('VAOS Page: ReasonForAppointmentPage', () => {
     });
   });
 
-  it.skip('should show alternate textbox char length if navigated via direct schedule flow', async () => {
+  it('should show alternate title if navigated via direct schedule flow', async () => {
```

Comment:
The product code logic does not vary textbox char length, it only varies the title depending on direct schedule or request.

Reply:
I think you can delete this test.  In the past, the textbox character limit differed between VA appointment direct schedule flow and Community Care request flow. Changes in the backend service updated so now they share the same character length limit.

This alternate title test has Community Care setup as a direct schedule. Unfortunately, this app does not have CC with direct schedule capability yet.

Reply:
Thanks, I'll remove it for now, but will this capability be added in the future?

Reply:
@JunTaoLuo, Not in the near horizon that I know of.